htdocs/fourn/facture/card.php

```diff
@@ -2287,9 +2287,15 @@
 						$("#socid").change(function() {
 							console.log("We have changed the company - Reload page");
 							// reload page
-							$("input[name=action]").val("create");
-							$("input[name=changecompany]").val("1");
-							$("form[name=add]").submit();
+							//$("input[name=action]").val("create");
+							//$("input[name=changecompany]").val("1");
+							//$("form[name=add]").submit();
+							var socid = $(this).val();
+							var projectid = "";
+							if ($(\'#projectid\').val() > 0) {
+								projectid = "&projectid="+$(\'#projectid\').val();
+							}
+							window.location.href = "'.$_SERVER["PHP_SELF"].'?action=create"+projectid+"&socid="+socid;
```

Comment:
If you reload this way, you will make a GET when we need a POST to not lose all fields already modified.
What is wrong with old code ?